code/src/Components/HappyThoughts.css

```diff
@@ -0,0 +1,46 @@
+.happy-thoughts {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    border: 1.5px solid black;
+    box-shadow: 7px 7px black;
+    width: 500px;
+    height: 100px;
+    margin: 20px;
+    padding: 20px;
+  }
```

Comment:
When the messages are long they don't fit in to the container any more. I think it's because you have a fixed height of 100px. A solution might be to remove the fixed height, or change the value to auto or fit-content. or if you want to have the same height for all messages, maybe change the height to something that works with the maximum amount of characters(140)? 
<img width="624" alt="Skärmavbild 2020-11-01 kl  21 02 27" src="https://user-images.githubusercontent.com/62887464/97873295-b69b0c80-1d17-11eb-9b34-20cf52c8e4c3.png">

Reply:
Thank you, yes, I am going to fix it!